lib/hexpm/web/templates/blog/002-organizations-going-live.html.md

```diff
@@ -0,0 +1,16 @@
+## Organizations going live
+
+<div class="subtitle">February 1, 2018 · by Eric Meadows-Jönsson</div>
+
+A few months ago we introduced the beta of private packages and organizations. In the meantime we
+have been building the billing system and iterating on organizations. We'd like to thank beta
+users for their valuable feedback.
+
+Today we are going out of beta and are going [live with organizations](/pricing). Beta users will
+have to go to the [organization dashboard](/dashboard) and enter a payment method and billing
+details to continue using organizations. You have **40 days** to enter a payment method.
```

Comment:
"You have **40 days** to enter a payment method." it is best to include the fixed date as well. that's because you probably mean 40 days from the time of publishing but it may not immediately click for everyone.